### PR TITLE
Since now on 4.x maintainance branch provides Java EE API version only

### PR DESCRIPTION
**Pull request by ropalka**
https://issues.redhat.com/browse/EJBCLIENT-440
https://issues.redhat.com/browse/EJBCLIENT-441
https://issues.redhat.com/browse/EJBCLIENT-442

### CODE REVIEW

**Comment by chengfang** [comment]
I'll need to fix the CI job setup so we can have valid test results.

**Comment by chengfang** [comment]
new versions of marshalling and naming-client are not available from maven central yet. This caused the build to fail.

File: pom.xml [hunk 1]
```diff
@@ -59,394 +53,313 @@
...
-                    </executions>
-                </plugin>
-            </plugins>
-        </pluginManagement>
+        <resources>
+            <resource>
+                <directory>src/main/java</directory>
+                <includes>
+                    <include>**/Version.properties</include>
+                </includes>
```
Comment (chengfang) [inline pom.xml:84]: the above <resource> element does not match any files. The version.properties is covered by the next <resource> element. Any reason it's added here?

**Comment by chengfang** [comment]
@ropalka  can you please rebase? I fixed CI setup and should give better results.

File: pom.xml (hunk 1 above)
Comment (ropalka) [reply pom.xml:84]: No specific reason. I used pom.xml from former JBoss EJB Client releases as a template. Fixed @chengfang

**Pull request merged**